sklearn/preprocessing/tests/test_common.py

```diff
@@ -81,7 +81,7 @@ def test_missing_value_handling(
         Xt_class = est.transform(X_train)
     kwargs = est.get_params()
     # remove the parameters which should be omitted because they
-    # are not defined in the sister function of the preprocessing class
+    # are not defined in the sibling function of the preprocessing class
```

Comment:
This change does not make sense here. I think that we can propose the following:

```suggestion
    # are not defined in the counterpart function of the preprocessing class
```